internal/attestation/renderer/chainloop/chainloop.go

```diff
@@ -177,18 +165,18 @@ func ExtractPredicate(envelope *dsse.Envelope) (NormalizablePredicate, error) {
 	// 2 - Extract the Chainloop predicate from the in-toto statement
 	switch statement.PredicateType {
 	case PredicateTypeV02:
-		var predicate *ProvenancePredicateV02
+		var predicate ProvenancePredicateV02
```

Comment:
out of curiosity, why replacing the pointers?

Reply:
I think at some point during my changes it needed to be preallocated. Not sure if that's still the case.